assets/sass/components/header.scss

```diff
@@ -1,17 +1,72 @@
 .header {
+  position: relative;
   align-items: center;
   display: flex;
   flex-wrap: wrap;
-  margin-bottom: 1.2em;
-  padding: 0.6em 2%;
+  margin-bottom: 4em;
+  padding: 0.6em 2% 0.6em 2%;
```

Comment:
Check `assets/sass/responsive.scss`, which takes care of the horizontal spaces for you. Add a `div` around the content of the header, and give that an `@include responsive-container();`. Media queries are already included in this.